### ISSUE TITLE
Reviewer 2 Comment 3

### ISSUE THREAD

**Issue by ramess101** [ramess101/JCED_FOMMS_Manuscript#27]
@mrshirts

> comment about equation 13: I am glad that the sign of the mu*N term is now corrected to be negative. I assumed the positive sign to be a misprint in the first MBAR paper. But when the wrong positive sign reappeared in subsequent publications I started to have doubt about whether some intricacy existed about the sign.

This is equation 13:

![image](https://user-images.githubusercontent.com/23408516/53427117-8d07da80-39a5-11e9-8fc2-f4253a9cf9cf.png)

@mrshirts I assume the "first MBAR paper" the reviewer is referring to is "Statistically optimal analysis of samples from multiple equilibrium states," where the reduced potential energy is defined as:

![image](https://user-images.githubusercontent.com/23408516/53427270-e243ec00-39a5-11e9-8b61-699d81988934.png)

I believe we have the sign correct in the present manuscript, but I just wanted to double check with you first.

**Comment by mrshirts** [comment]
Yeah, that's pretty stupid.  Somebody has been harboring a grudge. I wonder why they never wrote me directly.

**Comment by ramess101** [comment]
@mrshirts 

Yeah, I don't know why. Should we make a note of this to help avoid future confusion?

**Comment by mrshirts** [comment]
I'm working on a review, and I'll clarify there.

**Comment by ramess101** [comment]
@mrshirts 

Let me know if you want to modify my response:

![image](https://user-images.githubusercontent.com/23408516/53756373-35221580-3e76-11e9-853d-2eecaac3a13f.png)

**Comment by ramess101** [comment]
@mrshirts 

Could you check that you are OK with how I worded this response regarding the sign error?
